mirar/processors/base_catalog_xmatch_processor.py

```diff
@@ -158,7 +160,8 @@ def write_regions_files(self, image: Image, ref_cat: Table, img_cat: Table):
             unit=(u.deg, u.deg),
         )
 
-        clean_img_cat = self.sextractor_catalog_purifier(img_cat, image)
+        clean_img_cat, ref_cat = self.catalogs_purifier(img_cat, ref_cat, image)
+        logger.debug(clean_img_cat)
```

Comment:
What is this? Please add message if you want to leave this here

Reply:
reemoved